### PR TITLE
PLF-8383: fix news creation issues

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties [hunk 1]
```diff
@@ -7,10 +7,10 @@ activity.composer.news.illustration=Upload the image that will appear as illustr
 activity.composer.news.error.BrowserNotSupported=Your current browser does not seem to support file upload.
 activity.composer.news.error.TooManyFiles=You can only upload up to {0} files at once.
 activity.composer.news.error.FileTooLarge=File size exceeds the {0} MB limit.
-activity.composer.news.error.FileTypeNotAllowed=Erreur de traitement, type de fichier non autorisé.
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties]: Is it intended to remove this line ?

File: webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsFileDrop.vue [hunk 2]
```diff
@@ -14,7 +16,9 @@
       </div>
       <img v-if="files.length > 0" :src="files[0].src">
     </div>
-
+    <div v-show="error" class="uploadError alert alert-error v-content">
+      {{ error }}
+    </div>
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsFileDrop.vue]: This blocks already exists below. Is it intended to add it again ?

Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsFileDrop.vue]: removed

File: webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties (hunk 1 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties]: Done

File: webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsFileDrop.vue [hunk 3]
```diff
@@ -5,7 +5,9 @@
         <i class="uiIcon uiIconUpload"></i> {{ $t('activity.composer.news.label.attachFile') }}
       </label>
       <input id="uploadedFile" :value="selectedFile" type="file" class="attachFile"/>
-
+      <div v-show="error" class="uploadError alert alert-error v-content">	
+        {{ error }}	
+      </div>
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsFileDrop.vue]: Is this one also necessary (we have the same one below) ?

Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsFileDrop.vue]: Removed!

**Pull request merged**